### ISSUE TITLE
Support absolute path in URDF assets

### ISSUE THREAD

**Issue by michalpelka** [RobotecAI/o3de-ros2-gem#165]
**Is your feature request related to a problem? Please describe.**
xacro generates absolute path from xacro file:
Xacro :
```
 <mesh filename="file://$(find my_robot_description)/meshes/base_link.stl" scale="1 1 1"/>
```
URDF:
```
<mesh filename="file:///home/michal/ros2_ws/install/my_robot_description/share/my_robot_description/meshes/base_link.stl" scale="1 1 1"/>
```

**Describe the solution you'd like**
- I recommended to strip path to relative, since the o3de get assets from manager not from command line
- Or, simply provide the user with concise information that it is not supported.

**Comment by michalpelka** [comment]
This [PR](https://github.com/RobotecAI/o3de-ros2-gem/pull/156) should include solution for this issue.

**Comment by danielemorra98** [comment]
The engine doesn´t load any mesh. In the xacro file I tried both with the `"file://$(find my_robot_description)/meshes/base_link.stl` and with the `package://my_robot_description/meshes/base_link.stl` syntax. 


In the first case the **output** from the Editor Console is:
`[Error] (PrefabMakerUtils) - Could not find model asset for /home/daniele/ros2_ws/src/limo_description/urdf/file:///home/daniele/ros2_ws/src/limo_description/install/limo_description/share/limo_description/meshes/limo_wheel.dae`
In the second case the output is:
`[Error] (PrefabMakerUtils) - Could not find model asset for /home/daniele/ros2_ws/src/limo_description/urdf/limo_description/meshes/limo_wheel.dae`

The correct path should be `/home/daniele/ros2_ws/src/limo_description/meshes/limo_wheel.dae` (or `/home/daniele/ros2_ws/src/limo_description/install/limo_description/share/limo_description/meshes/limo_wheel.dae`)

**Comment by michalpelka** [comment]
Try to replace the absolute piece of the path with `package://` and put meshes to the Asset directory and allow to process by Asset Processor. 
At this moment we are thinking about an elegant way to solve this issue.

**Comment by danielemorra98** [comment]
I discovered that the prefix `package://` stands for `pwd `so I moved the URDF inside the project folder and the meshes inside Assets/O3DEScene/ (where there are the other .fbx files). I specified the new path for the meshes inside the URDF file.
With this new setup I don't get any more errors or warnings when loading the URDF but still no sign of meshes. 


REPO BRANCHES:
**ProjectRepo**
|
|__ myRobot.urdf
|
|__ **Assets/O3DEScene** __ myRobotMesh.dae

**Comment by michalpelka** [comment]
We merged #297 which has, a robust solution for discovering urdf<-> O3DE mapping of assets. It uses a checksum to find the correct mapping. I've tested the solution against multiple urdf and it works fine. Please check it out.
